OWNERS

```diff
@@ -3,8 +3,10 @@ approvers:
   - leodido
   - nibalizer
   - leogr
+  - cpanato
 reviewers:
   - Issif
   - leodido
   - nibalizer
   - leogr
+  - cpanato
```

Comment:
need to add a new line

also can order alphabetically?

Reply:
@cpanato Will do so in next change 😉 

@KeisukeYamashita I'll ask in next community, if everybody agrees like I do, I'll create the PR 👍, don't hesitate to come more often on Slack for chatting with everybody and discussing about changes.